tests/foreman/api/test_provisioningtemplate.py

```diff
@@ -26,60 +26,48 @@
 from fauxfactory import gen_mac
 from fauxfactory import gen_string
 from nailgun import client
-from nailgun import entities
 from requests.exceptions import HTTPError
 
-from robottelo.config import get_credentials
 from robottelo.config import settings
 from robottelo.config import user_nailgun_config
 from robottelo.utils.datafactory import invalid_names_list
 from robottelo.utils.datafactory import valid_data_list
 
 
-@pytest.fixture(scope="module")
-def module_location(module_location):
-    yield module_location
-    module_location.delete()
-
-
-@pytest.fixture(scope="module")
-def module_org(module_org):
-    yield module_org
-    module_org.delete()
-
-
-@pytest.fixture(scope="module")
-def module_user(module_org, module_location):
+@pytest.fixture(scope='module')
```

Comment:
Can we move this to some common module? module_user is something which can be used in other tests as well.

Reply:
this fixtures wouldn't be used in other component tests, will consider moving them there if we add any test related to it in UI/CLI, but I'd defer moving it there for now

Reply:
Totally optional but all fixtures present in one module would make easier to avoid duplication